srcpkgs/c-ares/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'c-ares'
 pkgname=c-ares
-version=1.16.1
+version=1.17.1
 revision=1
 build_style=gnu-configure
 short_desc="C library that performs DNS requests and name resolves asynchronously"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://c-ares.haxx.se/"
 distfiles="https://c-ares.haxx.se/download/${pkgname}-${version}.tar.gz"
-checksum=d08312d0ecc3bd48eee0a4cc0d2137c9f194e0a28de2028928c0f6cae85f86ce
+checksum=d73dd0f6de824afd407ce10750ea081af47eba52b8a6cb307d220131ad93fc40
 
 pre_configure() {
 	case "$CFLAGS" in
```

Comment:
You can remove this case and make it unconditional, I think. We always set `FORTIFY_SOURCE`